training/src/anemoi/training/losses/scalers/variable_level.py

```diff
@@ -81,62 +76,155 @@ def get_scaling_values(self, **_kwargs) -> torch.Tensor:
+            variable_level_scaling[idx] = self.get_level_scaling(variable_level)
 
         return variable_level_scaling
 
 
-class LinearVariableLevelScaler(BaseVariableLevelScaler):
+class BaseLinearVariableLevelScaler(BaseVariableLevelScaler):
+    """Configurable method converting variable level to loss scalings with a slope and intercept."""
+
+    def __init__(
+        self,
+        data_indices: IndexCollection,
+        group: str,
+        y_intercept: float,
+        slope: float,
+        metadata_extractor: ExtractVariableGroupAndLevel,
+        norm: str | None = None,
+        **kwargs,
+    ) -> None:
+        """Initialise variable level scaler.
+
+        Parameters
+        ----------
+        data_indices : IndexCollection
+            Collection of data indices.
+        group : str
+            Group of variables to scale.
+        y_intercept : float
+            Y-axis shift of scaling function.
+        slope : float
+            Slope of scaling function.
+        metadata_extractor : ExtractVariableGroupAndLevel
+            Metadata extractor for variable groups and levels.
+        norm : str, optional
+            Type of normalization to apply. Options are None, unit-sum, unit-mean and l1.
+        """
+        super().__init__(data_indices, group, metadata_extractor, norm)
+        del kwargs
+        self.y_intercept = y_intercept
+        self.slope = slope
+
+    def get_scaling_values(self, **_kwargs) -> torch.Tensor:
+        LOGGER.info("with slope = %s and y-intercept/minimum = %s.", self.slope, self.y_intercept)
+
+        return super().get_scaling_values(**_kwargs)
+
+
+class LinearVariableLevelScaler(BaseLinearVariableLevelScaler):
     """Linear with slope self.slope, yaxis shift by self.y_intercept."""
 
-    def get_level_scaling(self, variable_level: float) -> torch.Tensor:
+    def get_level_scaling(self, variable_level: float) -> float:
         return variable_level * self.slope + self.y_intercept
 
 
-class ReluVariableLevelScaler(BaseVariableLevelScaler):
+class ReluVariableLevelScaler(BaseLinearVariableLevelScaler):
     """Linear above self.y_intercept, taking constant value self.y_intercept below."""
 
-    def get_level_scaling(self, variable_level: float) -> torch.Tensor:
+    def get_level_scaling(self, variable_level: float) -> float:
         return max(self.y_intercept, variable_level * self.slope)
 
 
-class PolynomialVariableLevelScaler(BaseVariableLevelScaler):
+class PolynomialVariableLevelScaler(BaseLinearVariableLevelScaler):
     """Polynomial scaling, (slope * variable_level)^2, yaxis shift by self.y_intercept."""
 
-    def get_level_scaling(self, variable_level: float) -> torch.Tensor:
+    def get_level_scaling(self, variable_level: float) -> float:
         return (self.slope * variable_level) ** 2 + self.y_intercept
 
 
-class NoVariableLevelScaler(BaseVariableLevelScaler):
-    """Constant scaling by 1.0."""
+class ConstantLevelScaler(BaseVariableLevelScaler):
```

Comment:
I'm not sure the constant level scaler is needed? Because you could just use the novariablelevel scaler and then control the value of that scaler in the normal variable scalings?